test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperDescriptorFactoryTest.cs

```diff
@@ -1883,6 +1856,35 @@ public void ValidateTagHelperAttributeDescriptor_WithValidPrefix_ReturnsTrue(str
             return data;
         }
 
+        private static TagHelperDescriptor CreateTagHelperDescriptor(
```

Comment:
This refactoring removes coverage of a number of `TagHelperDescriptor` constructors. What is the upside?